src/app/window.ts

```diff
@@ -1163,10 +1237,12 @@ class XpraWindow {
       source_indication,
     ]);
     if (direction == MOVERESIZE_MOVE && mousedown_event) {
-      const e = mousedown_event;
+      const e = mousedown_event as any;
+      // TODO: Should not assign to an event
       e.type = "mousedown.draggable";
       e.target = this.div[0];
-      this.div.trigger(e);
+      // TODO: trigger is undefined -- this needs to be fixed
+      // this.div.trigger(e);
```

Comment:
I'm not sure what this is supposed to do -- this.div.trigger is consistently undefined throughout my test.

Reply:
oh, I think we need that - some windows trigger a window-move when you click on them.
We synthesize the event to trigger it when we get the message from the server.
You can find an example in `xpra toolbox`, it's called "Move-Resize". (click on move)

Reply:
Yeah, I'm not sure what's supposed to provide the trigger function? Is that a link via jQuery?

Reply:
Ah, right, it should be a jquery thing.
`div` was changed from a jquery object to a plain dom element, and here we need the jquery variant.